packages/generator-spectral/src/generators/formats/readers/openapi/actions.ts

```diff
@@ -68,6 +68,14 @@ const toGroupTag = (path: string): string =>
     (_, match) => toWords(match)
   );
 
+const toOperationName = (operationName: string): string =>
+  camelCase(
+    operationName.replace(
+      /^([0-9]+)(.+)$/,
+      (_, num, rest) => `${toWords(num)}${rest}`
```

Comment:
Will this potentially generate variable names with commas and spaces? 

```
// Large numbers:
converter.toWords(9007199254740992); // => “nine quadrillion, seven trillion, one hundred ninety-nine billion, two hundred fifty-four million, seven hundred forty thousand, nine hundred ninety-two”
```

Reply:
Maybe sufficient to wordify the leading number? Just remove the `+` in the regex?

Reply:
`camelCase` handles normalizing a lot of these cases but I can see where converting _all_ numbers is overkill. I'll reduce it to the single digit.